src/Plugin.NFC/Plugin.NFC.csproj

```diff
@@ -1,106 +1,90 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
-
-  <PropertyGroup>
-    <!--Work around so the conditions work below-->
-    <TargetFrameworks>netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid81;MonoAndroid90;MonoAndroid10.0;</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.16299</TargetFrameworks>
-    <!--Feel free to add as many targets as you need below
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why all these format changes? Can´t see the important change

Reply:
dunno. VS is driving me crazy. never touched this file by myself. replaced all tabs with spaces via vs for windows now